packages/rspack/src/executors/rspack/rspack.impl.ts

```diff
@@ -19,9 +19,29 @@ export default async function runExecutor(
   const compiler = await createCompiler(options, context);
 
   return new Promise<{ success: boolean }>((res) => {
-    compiler.run(() => {
-      compiler.close((err: any) => {
-        res({ success: !err });
+    compiler.run((error, stats) => {
+      // OURS
+      compiler.close(() => {
+        if (error) {
+          console.error(error);
+          res({ success: false });
+          return;
+        }
+        if (!compiler || !stats) {
+          res({ success: false });
+          return;
+        }
+
+        // TODO: Handle MultipleCompiler
```

Comment:
@jaysoo While debugging this with Jason we found that on the Rspack side they directly call the `rspack` function which internally also handles "multi compiler" (?) options. Worth giving it a look

https://github.dev/web-infra-dev/rspack/blob/706207a4922dff679ce6401f4368120031231d78/packages/rspack/src/rspack.ts#L1